docs/AboutUs.md

```diff
@@ -9,15 +9,14 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Kuan Jian Rong
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/jr.png" width="200px">
+
+[[github](https://github.com/kuanjr)]
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
```

Comment:
Perhaps you need to leave this link and have a skeleton project portfolio page

Reply:
I agree with Jack, i think best to link the github to your own github page and portfolio to your-github-username.md in the team folder.